include/rdma/fi_log.h

```diff
@@ -35,6 +35,8 @@
 #ifndef _FI_LOG_H_
 #define _FI_LOG_H_
 
+#include "config.h"
```

Comment:
Or why isn't is something like this (from other source files):

#if HAVE_CONFIG_H
#  include <config.h>
#endif /* HAVE_CONFIG_H */